code-samples/creating-your-bot/command-handling/commands/fun/beep.js

```diff
@@ -0,0 +1,10 @@
+const { SlashCommandBuilder } = require('discord.js');
+
+module.exports = {
+	data: new SlashCommandBuilder()
+		.setName('ping')
```

Comment:
```suggestion
		.setName('beep')
```
and the same here :)